.github/workflows/build-push-env-docker.yml

```diff
@@ -62,6 +62,6 @@ jobs:
           file: ./docker/pegasus-build-env/${{ matrix.dockertag }}/Dockerfile
           push: true
           tags: |
-            apache/pegasus:build-env-${{ matrix.dockertag }}
+            apache/pegasus:build-env-${{ matrix.dockertag }}-${{ github.ref_name }}
```

Comment:
https://github.com/apache/incubator-pegasus/blob/71eae1b7a2ec37cb88ee0f8530859fbb4a1c0380/docker/thirdparties-src/Dockerfile
and https://github.com/apache/incubator-pegasus/blob/71eae1b7a2ec37cb88ee0f8530859fbb4a1c0380/docker/thirdparties-bin/Dockerfile are using images like apache/pegasus:build-env-centos7, you should also consider this.

Reply:
OK, the images that are depending on build-env-* have also been updated.